pkg/crd/kubermatic/v1/cluster.go

```diff
@@ -533,6 +533,8 @@ type AzureCloudSpec struct {
 	RouteTableName    string `json:"routeTable"`
 	SecurityGroup     string `json:"securityGroup"`
 	AvailabilitySet   string `json:"availabilitySet"`
+	// LoadBalancerSKU sets the LB type that will be used for the Azure cluster, possible values are "basic" and "standard", if empty, "basic" will be used
+	LoadBalancerSKU string `json:"loadBalancerSKU"`
```

Comment:
Because we have clearly defined possible values, could you create a load balancer type for this? For instance
```
type LBSku string

const (
	LBSku StandardLBSku = "standard"
	LBSku BasicLBSku = "basic"
)
```

and later use it
```
LoadBalancerSku LBSku `json:"loadBalancerSku"`
```

later it's easier to use API and it is less prone to errors

Reply:
good point, ill add it